File: problemas/ad-hoc/2455/2455.jl
using Printf

entrada = [parse(Float64, x) for x in split(readline())]
p1 = entrada[1]
c1 = entrada[2]
p2 = entrada[3]
c2 = entrada[4]

diferenca = p2 * c2 - p1 * c1

if diferenca < 0
  @printf "-1\n"
else
  if diferenca == 0
    @printf "0\n"
  else
    @printf "1\n"
  end
end

File: problemas/ad-hoc/2455/2455.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI {
    static void Main(string[] args) {
        List<int> numeros = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();
        
        int diferenca = (numeros[2] * numeros[3]) - (numeros[0] * numeros[1]);
        
        if(diferenca < 0)       Console.WriteLine("-1");
        else if(diferenca > 0)  Console.WriteLine("1");
        else                    Console.WriteLine("0");
    }
}